bundles/org.openhab.binding.enocean/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -41,9 +46,15 @@
 				<label>Use following BaseId when connected directly to RS485 BUS</label>
 				<default>00000000</default>
 			</parameter>
-			<parameter name="nextSenderId" type="integer">
+			<parameter name="allowRepeatedLearn" type="boolean">
+				<advanced>true</advanced>
+				<label>Allow Repeated Learn Ins</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
Good point. Changed the meaning of this param to the opposite, should be more clearer now too.